arch/arm/boot/dts/bcm2711.dtsi

```diff
@@ -348,6 +348,8 @@
 			clock-names = "hdmi";
 			resets = <&dvp 0>;
 			ddc = <&ddc0>;
+			dmas = <&dma 10>;
```

Comment:
This patch "drm/vc4: HDMI audio support on Pi4. " and the revert will be removed. It was an intermediate step in cleaning up all the audio changes.